org.eclipse.winery.repository.rest/src/main/java/org/eclipse/winery/repository/rest/resources/admin/NamespacesResource.java

```diff
@@ -150,15 +152,21 @@ public String getPrefixForEncodedNamespace(@PathParam("namespace") String uri) {
 	 */
 	@GET
 	@Produces(MediaType.APPLICATION_JSON)
-	public List<NamespaceWithPrefix> getNamespacesAsJSONlist() {
+	public List<NamespaceWithPrefix> getNamespacesAsJSONlist(@QueryParam("all") String allNamespaces) {
 		Collection<Namespace> namespaces = this.getNamespaces();
 
 		// We now have all namespaces
 		// We need to convert from Namespace to String
 
 		List<NamespaceWithPrefix> namespacesList = new ArrayList<>();
 		for (Namespace ns : namespaces) {
-			namespacesList.add(new NamespaceWithPrefix(ns, this.namespaceManager.getPrefix(ns.getDecoded())));
+			if(allNamespaces != null){
```

Comment:
Please format using IntelliJ. Ctrl+Alt+L. --> Spaces missing

Please do the positive (non-negative!) branch first. Meaning `if (allNamespaces == null)`

Further `Objects.isNull` is preferred - https://docs.oracle.com/javase/8/docs/api/java/util/Objects.html#isNull-java.lang.Object- is preferred